src/util/constants.js

```diff
@@ -7,4 +7,6 @@ export const SORTING_OPTIONS = {
 export const HOMEPAGE_DOC_LAYOUTS = {
     list: "list",
     grid: "grid"
-}
\ No newline at end of file
+}
+
+export const DOC_ID_PARAM = "doc_id"
```

Comment:
it is hehe the path for the route looks like `/document/:doc_id` and then the actual URL looks like `/document/2`. I have it pulled out into the const file because we use it in multiple places (we need the param name to get the value)